### DISCUSSION TITLE
Multiple Roots - Block List Configu

### DISCUSSION THREAD

**Discussion by NikRimington** [KevinJump/uSync#660] [Q&A]
Hi,

I've got a scenario where I have 3 "root" folders (all configured in the appsettings.json file).

I have the definition of a block list in each of the root, but the top level root (i.e. changes for this specific site) don't appear to be working when it comes to block list.

At the base I have a block list that has 5 blocks defined in it
In the second root, I update some of the labels for the blocks
In the third root, I am removing 3 of blocks

After a uSync import the definitions from Base and Second root have merged correctly together, but the third root isn't being merged in.

This is what my appsettings looks like
```
"Settings": {
  "RootFolder": "uSync/corporate",
  "ImportOnFirstBoot": true,
  "IsRootSite": false,
  "Folders": [ "uSync/root", "uSync/v13", "uSync/corporate" ],
  "LockRoot": false
},
```

Root is a base platform,
v13 is a client specific base modification
corporate is a specific site

Running the site locally and making changes puts files in the corporate folder correctly, but when deployed to hosting the import doesn't apply them.

**Comment by KevinJump** [comment]
you're clearly are roots power user 💪 

I will take a look, I am guessing its the block merge stuff doing something not so cleve.
